### ISSUE TITLE
Add links from tables which don't interfere with selection

### ISSUE THREAD

**Issue by MytsV** [rucio/webui#582]
### Description

With enabling text selection in tables, links became less convenient. A way of having both links and redirects is putting the icon in front of the text and making only it clickable instead of the whole cell.
Addressing these issue, previously requested addition of links to all entities should be provided as well.

### Additional Information

- [ ] The Rucio API needs to be changed.
